src/geojson/Makefile

```diff
@@ -0,0 +1,332 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Generated files shouldn't be indexed and versioned.

Reply:
Yet ANOTHER merge issue.

Reply:
Should be gone. Also removed the binaries.